.eslintrc.js

```diff
@@ -16,6 +16,14 @@ module.exports = {
       plugins: ["@typescript-eslint", "jsx-a11y", "prettier"],
       rules: {
         "@typescript-eslint/await-thenable": "error",
+        "jsx-a11y/anchor-is-valid": [
```

Comment:
The Next/Link component oddly does not receive a classes but requires a child anchor link to customize the styles. But the link also requires a URL and errors if a URL is added on anchor. But an anchor without an href is invalid HTML. 
The below rule is used to work with (around) the Next/Link implementation.

Reply:
Have you tried using `passHref` on the Link component?
https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

Reply:
I tried with `passHref` and the rule still thinks it's an invalid anchor.  You're right we'll need to add an exception to the rule.